conformance/scenarios/httpgateway/httpgateway.yaml

```diff
@@ -0,0 +1,45 @@
+kind: Gateway
```

Comment:
Is it require, for conformance, that an implementation works with a minimal Gateway like this?

For Istio we have the GW declare which Service will be backing it. We have a default so it does technically work

Reply:
I think the simplest starting point for conformance is going to be to focus on these very minimal examples. This specific example seems like something we'd want everyone to support, but I haven't thought it through to carefully. If we run into issues here it can help us understand if there's problems with either the API, the implementations, or the tests.